content/blog/2019/corda-simple-broadcast-transaction/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Broadcasting a transaction to external organisations
+date: "2019-05-29"
+published: true
+tags: [corda, kotlin, dlt, distributed ledger technology, blockchain]
+cover_image: ./corda-concert.png
+github_url: https://github.com/lankydan/corda-broadcast-transaction
+---
+
+Corda does not globally broadcast transactions between organisations/nodes by default. Privacy and not sharing unnecessary data is a central component of Corda's design. Compared to other DLT (Distributed Ledger Technology) platforms and blockchains, this is indeed a big difference. But, this statement comes with one fallacy. There is a belief that Corda cannot broadcast transactions to external nodes at all. This is simply wrong. It may not be the default behaviour but it is definitely within Corda's capabilities. In this post, I will demonstrate the small amount of code required to send a transaction to any desired node.
+
+That being said, the code included in this post is for a trivial implementation. Moving forward, I aim to write further posts on this subject and move towards more complex and useful implementations for broadcasting transactions.
```

Comment:
i think it's worth while making a point that you can send anything across the wire with corda.. people think that they can only send signed transactions but you can send whatever object you like as long as it is serializable!

Reply:
somewhat address in other comment